### PR TITLE
update operator instructions

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 1]
```diff
@@ -79,13 +79,63 @@ FIXME update with the Marketplace ECR
...
+
+[source,shell script]
+----
+helm repo add akka-operator-helm https://lightbend.github.io/akka-operator-helm/
+helm repo update
+----
+
+[source,shell script]
+----
+helm install akka-operator akka-operator-helm/akka-operator-<version>.tgz --namespace lightbend --set image.repository=803424716218.dkr.ecr.eu-central-1.amazonaws.com/akka-operator --set image.tag=<version> --create-namespace
```
Comment (patriknw) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:94]: I see that Cloudflow is using `--version "2.0.22"` parameter. Can that replace both `<version>` in above command?

Comment (andreaTP) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:94]: After https://github.com/lightbend/akka-platform-operator/pull/324 get merged yes.
The versions are kept uniform across helm / docker etc.
The updated command looks like this:
https://github.com/lightbend/akka-platform-operator/pull/324/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R60-R64

Comment (andreaTP) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:94]: should we have the `image.repository` default to `803424716218.dkr.ecr.eu-central-1.amazonaws.com/akka-operator` in the Helm chart itself? Looks like a good default at this point in time, isn't it?

Comment (patriknw) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:94]: that would be good, but depends on how it works with with the real product ECR (is it same for all users?), @seglo might (soon) know more?

Comment (seglo) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:94]: End users will use the production ECR coordinates, which don't exist yet. We can leave this for now for internal testing and update it when the production ECR is available.

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 2]
```diff
@@ -79,13 +79,63 @@ FIXME update with the Marketplace ECR
...
 ----
-helm install akka-operator target/akka-operator-<version>.tgz --namespace lightbend --set image.repository=803424716218.dkr.ecr.eu-central-1.amazonaws.com/akka-operator --set image.tag=<tag> --create-namespace
+aws ecr get-login-password --region eu-central-1 | docker login --username AWS --password-stdin 803424716218.dkr.ecr.eu-central-1.amazonaws.com
+----
+
+Update the Akka Microservices CRD (replace the `<version>` in the URL):
+
+[source,shell script]
+----
+kubectl apply -f https://raw.githubusercontent.com/lightbend/akka-operator-helm/master/akka-operator/crds/<version>/akka-microservices-crd.yml
```
Comment (patriknw) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:123]: I guess this should be https://lightbend.github.io/akka-operator-helm/akka-operator/crds/v1/akka-microservices-crd.yml

`s/v1/<version>/`

right @andreaTP ?

File: docs-source/docs/modules/deployment/pages/aws-install.adoc (hunk 1 above)
Comment (seglo) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:94]: You can update this for the staging ECR if you want. I also created a PR to update to the staging ECR in the helm chart itself, so once that's merged you could just omit the `image.repository` setting altogether in the guide.

```suggestion
helm install akka-operator akka-operator-helm/akka-operator-<version>.tgz --namespace lightbend --set image.repository=709825985650.dkr.ecr.us-east-1.amazonaws.com/lightbend/akka-operator --set image.tag=<version> --create-namespace
```

Comment (seglo) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:94]: https://github.com/lightbend/akka-operator-helm/pull/5

Comment (patriknw) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:94]: thanks

Comment (patriknw) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc]: I haven't  changed  the coordinates  yet, let's do that in separate PR

Comment (patriknw) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc]: changed to `helm install akka-operator akka-operator-helm/charts/akka-operator-<version>.tgz`

**Review by seglo — COMMENTED** [review]
Forgot to hit the submit button yesterday. Maybe we can make a note about picking the right version?

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 3]
```diff
@@ -79,13 +79,63 @@ FIXME update with the Marketplace ECR
...
+Update the Akka Platform Operator with Helm.
+
+Add the Akka Platform Operator Helm repository and update the local index:
+
+[source,shell script]
+----
+helm repo add akka-operator-helm https://lightbend.github.io/akka-operator-helm/
+helm repo update
+----
+
```
[the remark this answers is not in the document]
Comment (patriknw) [reply docs-source/docs/modules/deployment/pages/aws-install.adoc:205]: @seglo sounds good, please PR

**Pull request merged**